recipes-bsp/u-boot/patches/0001-configs-clearfog-Mender-integration.patch

```diff
@@ -0,0 +1,37 @@
+From d494f2cb0e02c03f227a98afc63e0fcacd8e7b2c Mon Sep 17 00:00:00 2001
+From: Mirza Krak <mirza.krak@northern.tech>
+Date: Mon, 11 Mar 2019 09:05:37 +0000
+Subject: [PATCH 1/1] configs: clearfog: Mender integration
+
+Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
+---
+ include/configs/clearfog.h | 11 ++---------
+ 1 file changed, 2 insertions(+), 9 deletions(-)
+
+diff --git a/include/configs/clearfog.h b/include/configs/clearfog.h
+index 6c74d249e1..ea278d74bf 100644
+--- a/include/configs/clearfog.h
++++ b/include/configs/clearfog.h
+@@ -52,17 +52,10 @@
+ /* Environment Organization */
+ #define CONFIG_ENV_SECT_SIZE		0x200
+ #define CONFIG_ENV_SIZE			(8 * 1024)
+-#define CONFIG_ENV_OFFSET		(SZ_1M - CONFIG_ENV_SIZE)
+ #define CONFIG_ENV_ADDR			CONFIG_ENV_OFFSET
+ 
+-#ifdef CONFIG_ENV_IS_IN_MMC
+-#ifdef CONFIG_MVEBU_SPL_BOOT_DEVICE_MMC
+-#define CONFIG_SYS_MMC_ENV_DEV          0
+-#define CONFIG_SYS_MMC_ENV_PART         1
+-#else
+-#define CONFIG_SYS_MMC_ENV_DEV          0
+-#endif
+-#endif
++#define CONFIG_BOOTCOUNT_LIMIT
++#define CONFIG_BOOTCOUNT_ENV
+ 
+ #if defined(CONFIG_CMD_SCSI) || defined(CONFIG_CMD_BUBT) || defined(CONFIG_ENV_IS_IN_SATA)
+ #define CONFIG_SYS_SATA_ENV_DEV		0
+-- 
+2.11.0
+
```

Comment:
You seem to be adding the Mender files here, which where removed in #4

Reply:
Yes, my intention was for this patch to be applied first. Hopefully it's suitable now.